<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Privacy</class>
 <widget class="QWidget" name="Privacy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="gb_settings">
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QGridLayout">
      <item row="1" column="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cb_default">
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Default List (all sessions):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Active List (current session):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cb_active">
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_listSettings">
     <property name="title">
      <string>List Editor</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>List:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cb_lists">
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::AdjustToContents</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pb_newList">
          <property name="text">
           <string>New List</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_renameList">
          <property name="text">
           <string>Rename List</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_deleteList">
          <property name="text">
           <string>Delete List</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="ck_autoActivate">
        <property name="text">
         <string>Automatically activate this list on connect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gb_rules">
        <property name="title">
         <string>Rules</string>
        </property>
        <layout class="QVBoxLayout">
         <item>
          <widget class="QListView" name="lv_rules"/>
         </item>
         <item>
          <layout class="QHBoxLayout">
           <item>
            <widget class="QPushButton" name="pb_add">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pb_remove">
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pb_up">
             <property name="text">
              <string>Up</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pb_down">
             <property name="text">
              <string>Down</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pb_edit">
             <property name="text">
              <string>Edit ...</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>111</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pb_apply">
             <property name="text">
              <string>Apply</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cb_active</tabstop>
  <tabstop>cb_default</tabstop>
  <tabstop>cb_lists</tabstop>
  <tabstop>pb_newList</tabstop>
  <tabstop>pb_deleteList</tabstop>
  <tabstop>ck_autoActivate</tabstop>
  <tabstop>lv_rules</tabstop>
  <tabstop>pb_add</tabstop>
  <tabstop>pb_remove</tabstop>
  <tabstop>pb_up</tabstop>
  <tabstop>pb_down</tabstop>
  <tabstop>pb_edit</tabstop>
  <tabstop>pb_apply</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
